src/main/resources/db/migration/V1_49_1__contract_type_data.sql

```diff
@@ -0,0 +1,20 @@
+insert into contract_type(id, name)
+Values  ('72e60faf-b8e5-4699-9d7c-aef631cca71b', 'Accomodation'),
+        ('b402486d-991e-4977-9291-073a3526d60f', 'Education, Training and Employment'),
```

Comment:
should these names match the other service cat names exactly e.g. 

```
Education, Training and Employment (ETE)
Finance, Benefits and Debt (FBD)
Dependency and Recovery (D&R)
```

Reply:
well, it would be best to keep them to the level procurement knows -- to make life easier outside our boundaries